orchagent/aclorch.cpp

```diff
@@ -5112,3 +5168,48 @@ void AclOrch::removeAllAclRuleStatus()
     }
 }
 
+// Bind egress ACL table (with bind type switch) to switch
+bool AclOrch::bindEgrAclTableToSwitch(AclTable &table)
+{
+    sai_attribute_t attr;
+    attr.id = SAI_SWITCH_ATTR_EGRESS_ACL;
+    attr.value.oid = table.getOid();
+
+    sai_status_t status = sai_switch_api->set_switch_attribute(gSwitchId, &attr);
+    if (status == SAI_STATUS_SUCCESS)
+    {
+        table.bindToSwitch = true;
+        SWSS_LOG_NOTICE("Bind egress acl table %s to switch", table.id.c_str());
+        return true;
+    }
+    else
+    {
+       SWSS_LOG_ERROR("Failed to bind egress acl table %s to switch", table.id.c_str());
+       return false;
+    }
+}
+
+// Unbind egress ACL table from swtich
+bool AclOrch::unbindEgrAclTableFromSwitch(AclTable &table)
```

Comment:
Same as bind aclTable, please use ACL Group instead of ACLTable and switchorch already has the code.

Reply:
Same as above.